src/canvas-force-graph.js

```diff
@@ -61,6 +61,7 @@ export default Kapsule({
     linkDirectionalParticleWidth: { default: 4, triggerUpdate: false },
     linkDirectionalParticleColor: { triggerUpdate: false },
     globalScale: { default: 1, triggerUpdate: false },
+    d3AlphaMin: { default: 0, triggerUpdate: false, onChange(alphaMin, state) { state.forceLayout.alphaMin(alphaMin) }},
```

Comment:
As in the [three-forcegraph commit](https://github.com/vasturiano/three-forcegraph/commit/9469a3f53a5451288c5cd3784aa13511e777abac), the `onChange` part isn't needed, because we're doing the checking of the alphaMin outside of the force engine.